AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -111,12 +111,12 @@ + (BOOL)tableExists:(NSString *)tableName inOpenedDatabase:(void *)db {
   NSString *query =
       [NSString stringWithFormat:@"SELECT COUNT(*) FROM \"sqlite_master\" WHERE \"type\"='table' AND \"name\"='%@';", tableName];
   NSArray<NSArray *> *result = [MSDBStorage executeSelectionQuery:query inOpenedDatabase:db];
-  return (result.count > 0) ? [(NSNumber *)result[0][0] boolValue] : NO;
+  return result.count > 0 && result[0].count > 0 ? [(NSNumber *)result[0][0] boolValue] : NO;
```

Comment:
Can we use `querySingleValue` here?

Reply:
It's returning different type. I guess that's why @dtrabo didn't change this one.